net/tn5250/Portfile

```diff
@@ -1,33 +1,29 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           github 1.0
 
+github.setup        tn5250 tn5250 0.17.6 v
+github.tarball_from releases
 name                tn5250
```

Comment:
The "name" is set automatically from the "github.setup" li e so can be removed here